README.md

```diff
@@ -45,3 +58,10 @@ The auction system is the sum of 5 micro-services and a web gateway:
 * **web-gateway**: a [Play](https://www.playframework.com/) application providing web UI and acting as gateway to all previously described services.
 
 Check the docs for each service for details on the **commands** and **queries** it serves as well as events the service **emits** and events it **consumes** from the [Message Broker](http://www.lagomframework.com/documentation/1.3.x/java/MessageBroker.html#Message-Broker-Support).
+
+## Good to know
+
+- Hello World Giter8 seed
```

Comment:
This title doesn't make sense. In java either sbt's g8 or mvn archetypes could be used.